frontend/lib/src/components/widgets/AudioInput/AudioInput.tsx

```diff
@@ -409,10 +407,15 @@ const AudioInput: React.FC<Props> = ({
   const isDisabled = disabled || hasNoMicPermissions
 
   return (
-    <StyledAudioInputContainerDiv
-      className="stAudioInput"
-      data-testid="stAudioInput"
-    >
+    <>
+      {deleteFileUrl && (
+        <ToolbarAction
+          label="Clear recording"
+          icon={Delete}
+          onClick={() => handleClear({ updateWidgetManager: true })}
+          data-testid="stAudioInputClearRecordingButton"
```

Comment:
**nit:** I recognize this is just moving code, but still leaving this feedback here. We should move away from data-testids except for the cases where they are actually really required. It's preferred to utilize accessible selectors instead.

Reply:
good call, i don't think that's even being used anymore and can be cleaned up

Reply:
cleaned this up in the download button #9637